### DISCUSSION TITLE
Github Pages tutorial creates some confusions for beginners

### DISCUSSION THREAD

**Discussion by dkrasne** [skills/.github#105] [Q&A]
Here are a few hitches I ran into while setting up GitHub Pages from the tutorial:

1. Under 'how to start this course,' the screenshot is from introduction-to-github, not github-pages. It's not clear whether one should choose the name of one's future blog address, or if it makes sense to leave the default name of github-pages. This should be clarified.
2. Step 1, instruction 3 ("Click Pages, in the "GitHub Pages" section, use the Source drop-down, then select main branch.") did not match exactly what I saw. 'Main branch' was not an option under the Source drop-down; the only option other than GitHub Actions was "Deploy from a Branch". The **SECOND** drop-down menu (under Branch, not Source) gave "main" as an option.
3. I'm concerned that because of one or both of these unclear instructions, I did something wrong at some point, because when I go to my new page (currently at https://dkrasne.github.io/github-pages/ although I plan to rename it if I can't figure this out), the bottom of the page reads: "github-pages" and "Lesson on building a blog with Jekyll". So I think I _must_ have done something wrong (especially since I had to search for the button to merge the pull request, it didn't show up automatically), but I can't figure out where, since I followed all of the subsequent instructions precisely.
4. Even if I actually did everything correctly, those two unclear instructions should be changed, for the sake of beginners.

Help would be appreciated, thanks!

**Comment by sinsukehlab** [comment]
Here are my responses.
1. I agree. I have already opened an issue skills/template-template#21 from a discussion #66.
1. The instruction should be updated because the UI of GitHub Pages has now been updated. I have just opened a pull request skills/github-pages#18.
    Cf. https://pages.github.com/
1. You just have to go to the **Pull requests** tab to merge a pull request.

I would appreciate it if you could mark this suggest as the answer.
Please feel free to ask any further questions.

**Reply by dkrasne** [reply]
Thanks; my issue in point 3 wasn't that I couldn't find the place to merge the pull request (I did find it), but that it's supposed to show up automatically if I've followed all the instructions correctly (at least, that's what happened when I did the original GitHub tutorial, and this tutorial claims the same thing will happen). It didn't show up automatically, and as I say, the bottom of my new pages still reads: "github-pages" and "Lesson on building a blog with Jekyll," which is obviously not supposed to be the case. :)

**Reply by dkrasne** [reply]
Also, my primary question in point 1 is: It's not clear whether one should choose the name of one's future blog address, or if it makes sense to leave the default name of github-pages. Thanks!

**Reply by sinsukehlab** [reply]
> Thanks; my issue in point 3 wasn't that I couldn't find the place to merge the pull request (I did find it), but that it's supposed to show up automatically if I've followed all the instructions correctly (at least, that's what happened when I did the original GitHub tutorial, and this tutorial claims the same thing will happen). It didn't show up automatically, and as I say, the bottom of my new pages still reads: "github-pages" and "Lesson on building a blog with Jekyll," which is obviously not supposed to be the case. :)

I couldn't understand what you meant by "automatically". Could you show me a screenshot?
Did you mean that a button to merge a pull request should show up at [your GitHub Pages site](https://dkrasne.github.io/github-pages/)? That would not happen because your site is not a GitHub repository.
You should wait ~20 seconds and then refresh the tutorial repository.

**Reply by sinsukehlab** [reply]
> Also, my primary question in point 1 is: It's not clear whether one should choose the name of one's future blog address, or if it makes sense to leave the default name of github-pages. Thanks!

I highly recommend you **not** to name a generated repository "github-pages" or "introduction-to-github", for those names  are used for forked repositories.
GitHub says, "Great repository names are short and memorable. Need inspiration? How about [random-name]?" when you generate a repository. Just using this random name is fine. Otherwise you can choose your blog name or whatever but "github-pages".
